src/services/storageProvider/hooks/index.js

```diff
@@ -0,0 +1,50 @@
+const CryptoJS = require('crypto-js');
+const { authenticate } = require('@feathersjs/authentication');
+
+const globalHooks = require('../../../hooks');
+
+const encryptSecret = (context) => {
+	if (context.data.secretAccessKey) {
+		context.data.secretAccessKey = CryptoJS.AES
+			.encrypt(context.data.secretAccessKey, process.env.S3_KEY)
+			.toString();
+	}
+	return context;
+};
+
+const decryptSecret = (context) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['decryptSecret' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5222818)